crates/stackable-operator/src/builder/pod/volume.rs

```diff
@@ -358,9 +358,9 @@ impl SecretOperatorVolumeSourceBuilder {
                 metadata: Some(ObjectMetaBuilder::new().annotations(annotations).build()),
                 spec: PersistentVolumeClaimSpec {
                     storage_class_name: Some("secrets.stackable.tech".to_string()),
-                    resources: Some(ResourceRequirements {
+                    resources: Some(VolumeResourceRequirements {
```

Comment:
question: This is a non-breaking change for operator users, but a breaking one for library users? The type changes but the (de)serialization is the same, right?

We should mark the changelog entry as breaking in my opinion.

Reply:
I mean technically we (as operator-rs) are not making a breaking change and I think we do not re-export k8s-openapi. But I'll add a changelog entry anyways :)